components/x-eventpromo/src/Image.jsx

```diff
@@ -0,0 +1,25 @@
+import { h } from '@financial-times/x-engine';
+
+const Image = ({ linkUrl, imageUrl }) => {
+	const baseSrc = `https://www.ft.com/__origami/service/image/v2/images/raw/${imageUrl}?source=next&amp;fit=cover&amp;compression=best&amp;`;
+	const imgSrc = `${baseSrc}width=340 340w,${baseSrc}width=400 740w`;
+
+	return (
+		<a href={linkUrl}
+						className="js-event-promo--animate"
+						data-trackable="event-promo"
+						aria-hidden="true"
+						tabIndex="-1"
+		>
+			<img
+				className="js-event-promo--animate event-promo-inarticle__img fade-0"
+				srcSet={imgSrc} alt=""
```

Comment:
alt needs a value probably eventTitle and possibly the strapline

Reply:
The image is marked as for presentation only so blank alt text is recommended, see https://www.w3.org/WAI/tutorials/images/decorative/